### DISCUSSION TITLE
Cookie “PHPSESSID” does not have a proper “SameSite” attribute value

### DISCUSSION THREAD

**Discussion by f1-outsourcing** [phalcon/cphalcon#16626] [General]
I am new with phalcon and don't really get how to set options to the default php session and how this session start relates to the options array for `$this->cookies->set`

```
    $di->setShared('session', function () use ($config){
        //$session = new \Phalcon\Session\Adapter\Files();
        $session = new Manager();
        $session->setOptions(["samesite" => "Strict" ]);
        $file = new \Phalcon\Session\Adapter\Stream(array('savePath'=>$config->tempDir));
        $session->setAdapter($file)->start();
        return $session;
    });
```
In this stackoverflow example[1] they are setting the ini. I prefer to skip setting values ini.

[1]
https://stackoverflow.com/questions/64442252/session-lose-on-bank-redirect-phalconphp

**Comment by niden** [comment]
The session manager or any adapter are not setting any "session" cookies or their properties - it is all done internally via the PHP session manager. 

If you want to change any of those settings your best bet is to use `ini_set` as per the SO example.
